### PR TITLE
Add nodeSelector and annotations to some helm templates.  (#7339) (#7345)

### PR DESCRIPTION
**Pull request by PFedak**

### CODE REVIEW

File: etc/helm/pachyderm/templates/console/service.yaml [hunk 1]
```diff
@@ -6,6 +6,7 @@ SPDX-License-Identifier: Apache-2.0
 apiVersion: v1
 kind: Service
 metadata:
+  annotations: {{- toYaml .Values.console.service.annotations | nindent 4 }}
```
Comment (chainlink) [inline etc/helm/pachyderm/templates/console/service.yaml:9]: We probably should surround this in an if statement

**Review by chainlink — APPROVED** [review]
Looks good with one small edit

File: etc/helm/pachyderm/templates/console/service.yaml (hunk 1 above)
Comment (PFedak) [reply etc/helm/pachyderm/templates/console/service.yaml:9]: ah, fair, I had seen some places where we let the bare `{}` through but that seems fine

Comment (chainlink) [reply etc/helm/pachyderm/templates/console/service.yaml:9]: It might work, generally safer to go with the if though

**Pull request merged**